easybuild/easyconfigs/p/PSORTb/PSORTb-3.0.4-libpsortb.patch

```diff
@@ -0,0 +1,128 @@
+diff -ru bio-tools-psort-all/lib/Bio/Tools/PSort/Install.pm bio-tools-psort-all.new/lib/Bio/Tools/PSort/Install.pm
```

Comment:
please include a comment in the patch explaining what it does, and who the author is